tests/migration_engine/conftest.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+from colorama import Style
+import pytest
+
+from skibidi_orm.migration_engine.db_config.base_config import (
+    BaseDbConfig,
+    ConfigSingleton,
+)
+
+
+@pytest.fixture(autouse=True)
+def reset_config_singleton(monkeypatch: pytest.MonkeyPatch):
+    monkeypatch.setattr(ConfigSingleton, "_instances", {})
+    monkeypatch.setattr(BaseDbConfig, "_BaseDbConfig__instances_count", 0)
+
+
+@pytest.hookimpl(hookwrapper=True)
+def pytest_runtest_makereport(item: Any) -> Any:
```

Comment:
could you document this? i don't how exactly this functions aside from wrapping a test result? what types do we expect the outcome and item to be and why is the attribute called nodeid?